runner/src/main/java/org/ananas/runner/steprunner/sql/SQLTransformer.java

```diff
@@ -32,6 +36,8 @@ public void build() {
                     .registerUdf("zeroIfNull", new NullableBigDecimalFn())
                     .registerUdf("zeroIfNull", new NullableIntegerFn())
                     .registerUdf("emptyIfNull", new NullableStringFn())
-                    .registerUdf("hash", new HashFn()));
+                    .registerUdf("hash", new HashFn())
```

Comment:
Please rename these UDF with capital letters and '_'

Reply:
It's capitalized now.